lib/Doctrine/AuditLog/Listener.php

```diff
@@ -100,6 +100,8 @@ public function preDelete(Doctrine_Event $event)
 	        $event->getInvoker()->set($name, null);
 
             if ($this->_auditLog->getOption('deleteVersions')) {
+                $conditions = array();
+                $values = array();
```

Comment:
Same here, defining the arrays in the same scope that they end up getting used later.